converter/generate_image.py

```diff
@@ -6,7 +6,7 @@
 @bp.route('/get-image/')
 def generate_image():
     try:
-        return send_file(converter.parse('result.abc').write('musicxml.png'), 
+        return send_file(converter.parse('musicxmlCache/ActorPreludeSample.musicxml').write('musicxml.png'), 
```

Comment:
I don't understand why are we hardcoding the file path here?  Why `'musicxmlCache/ActorPreludeSample.musicxml'`?

Reply:
was for testing. Corrected